packages/strapi-parts/src/Card/Card.stories.mdx

```diff
@@ -27,7 +28,18 @@ This is the doc of the `Card` component
 
 ## Card
 
-Description...
+Cards are used to gather a set of information that needs to stand out from the rest of the page. 
+
+**Best practices**
+
+- Content within the Card is mostly optional. Pick up what is needed.
+- Content can be added to the Card. Keep in mind that spacings are important in a Card component.
+- Main text is truncated at the end of the first line.
```

Comment:
This has not been implemented 🤔

Reply:
And we need to define how to access the "hidden" information if we want this (+ spec this)

Reply:
Currently you access the "hidden" info via a tooltip